src/Components/Facility/InventoryList.tsx

```diff
@@ -1,65 +1,36 @@
-import { useState, useCallback, useEffect, lazy } from "react";
-
+import { useState, lazy } from "react";
 import { navigate } from "raviger";
-import { useDispatch } from "react-redux";
-import { statusType, useAbortableEffect } from "../../Common/utils";
-import { getInventorySummary, getAnyFacility } from "../../Redux/actions";
 import Pagination from "../Common/Pagination";
 import { classNames } from "../../Utils/utils";
 import Page from "../Common/components/Page";
 import ButtonV2 from "../Common/components/ButtonV2";
 import { NonReadOnlyUsers } from "../../Utils/AuthorizeFor";
+import useQuery from "../../Utils/request/useQuery";
+import routes from "../../Redux/api";
 const Loading = lazy(() => import("../Common/Loading"));
 
 export default function InventoryList(props: any) {
   const { facilityId }: any = props;
-  const dispatchAction: any = useDispatch();
-  const [isLoading, setIsLoading] = useState(false);
-  const initialInventory: any[] = [];
   let inventoryItem: any = null;
-  const [inventory, setInventory] = useState(initialInventory);
   const [offset, setOffset] = useState(0);
   const [currentPage, setCurrentPage] = useState(1);
-  const [totalCount, setTotalCount] = useState(0);
-  const [facilityName, setFacilityName] = useState("");
   const limit = 14;
 
-  const fetchData = useCallback(
-    async (status: statusType) => {
-      setIsLoading(true);
-      const res = await dispatchAction(
-        getInventorySummary(facilityId, { limit, offset })
-      );
-      if (!status.aborted) {
-        if (res?.data) {
-          setInventory(res.data.results);
-          setTotalCount(res.data.count);
-        }
-        setIsLoading(false);
-      }
+  const { data: InventotyData } = useQuery(routes.getInventorySummary, {
```

Comment:
1. It's a good practice to use proper naming conventions for variables. Use camelCase instead of PascalCase.
2. Fix the spelling mistakes.

```suggestion
  const { data: inventoryData } = useQuery(routes.getInventorySummary, {
```